release/k8s_tests/run_gcs_ft_on_k8s.py

```diff
@@ -339,47 +354,45 @@ def get_stats():
 
 
 def main():
-    procs = []
-    try:
-        check_kuberay_installed()
-        start_rayservice()
-        procs.append(start_port_forward())
-        warmup_cluster(200)
-        users = 60
-        duration = 5 * 60 * 60
-        procs.append(start_sending_traffics(duration * 1.1, users))
-        start_killing_nodes(duration, 60, 6)
-        rate, qps, data = get_stats()
-
-        print("Result:", rate, qps)
-
+    result = {KILL_WORKER_NODE: {"rate": None}, KILL_HEAD_NODE: {"rate": None}}
+    expected_result = {KILL_WORKER_NODE: 0.99, KILL_HEAD_NODE: 0.99}
+    check_kuberay_installed()
+    users = 60
+    for kill_node_type, kill_interval, test_duration in [
+        (KILL_WORKER_NODE, 60, 600),
+        (KILL_HEAD_NODE, 300, 1200),
+    ]:
         try:
-            assert rate > 0.999
-            assert qps > users * 10 * 0.8
-        except Exception:
-            print("Raw Data", data)
-            print("Result:", rate, qps)
-            raise
-
-    except Exception as e:
-        print("Experiment failed")
-        raise e
-    finally:
-        print("=== Cleanup ===")
-
-        subprocess.run(
-            ["kubectl", "delete", "-f", str(yaml_path)],
-            capture_output=True,
-        )
+            generate_cluster_variable()
+            procs = []
+            start_rayservice()
+            procs.append(start_port_forward())
+            warmup_cluster(200)
+            procs.append(start_sending_traffics(test_duration * 1.1, users))
+            start_killing_nodes(test_duration, kill_interval, kill_node_type)
+            rate, qps, _ = get_stats()
+            result[kill_node_type]["rate"] = rate
+            assert expected_result[kill_node_type] <= rate
+        except Exception as e:
+            print(f"{kill_node_type} HA test failed, {e}")
+        finally:
+            print("=== Cleanup ===")
+            subprocess.run(
+                ["kubectl", "delete", "-f", str(yaml_path)],
+                capture_output=True,
+            )
+            subprocess.run(
+                ["helm", "uninstall", locust_id],
+                capture_output=True,
+            )
 
-        subprocess.run(
-            ["helm", "uninstall", locust_id],
-            capture_output=True,
-        )
+            print("Kill locust processes")
+            for p in procs:
+                p.kill()
+        print("Result:", result)
 
-        print("Kill processes")
-        for p in procs:
-            p.kill()
+        with open("/tmp/serve_kuberay_ha.json", "wt") as f:
```

Comment:
Consider using `TEST_OUTPUT_JSON` like the other release tests, I'm not completely sure what it does but it might make debugging easier (It might make the output appear in buildkite directly or persist the results somewhere)